install/install.php

```diff
@@ -327,24 +331,27 @@ public function __construct($dbh)
         allow_signed_keys: false
     );
 
+    /** @var array $CFG_GLPI */
+    global $CFG_GLPI;
+    $CFG_GLPI['url_base'] = \Symfony\Component\HttpFoundation\Request::createFromGlobals()->getBasePath();
+
     if ($success) {
-        echo "<p>" . __s('Initializing database tables and default data...') . "</p>";
-        try {
-            Toolbox::createSchema($_SESSION["glpilanguage"]);
-        } catch (\Throwable $e) {
-            echo "<p>"
-                . sprintf(
-                    __s('An error occurred during the database initialization. The error was: %s'),
-                    '<br />' . htmlescape($e->getMessage())
-                )
-                . "</p>";
-            @unlink(GLPI_CONFIG_DIR . '/config_db.php'); // try to remove the config file, to be able to restart the process
-            $prev_form($host, $user, $password);
-            return;
-        }
-        echo "<p>" . __s('OK - database was initialized') . "</p>";
+        echo "<p>" . __('Initializing database tables and default data...') . "</p>";
+
+        echo \sprintf('<p>%s :</p>', __('Progress'));
+
+        echo '<div id="glpi_install_messages_container"></div>';
 
+        echo '<div id="glpi_install_success" class="container">';
         $next_form();
+        echo '</div>';
+
+        $prev_form($host, $user, $password);
```

Comment:
IMHO, the `Back` button should be disabled too.

Also, there is a styling issue, but it is not related to this PR. It is supposed to be displayed this way:
![image](https://github.com/user-attachments/assets/b04c19ff-1125-42e3-a61c-9f325cb91ecc)